### PR TITLE
update plugins in docs and correct things in index.md

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**- What I did**
1. update plugin docs
2. correct things in index.md

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/commandline/index.md [hunk 1]
```diff
@@ -62,20 +62,20 @@ read the [`dockerd`](dockerd.md) reference page.
 | [events](events.md) | Get real time events from the server                   |
 | [exec](exec.md) | Run a command in a running container                       |
 | [export](export.md) | Export a container's filesystem as a tar archive       |
-| [kill](kill.md) | Kill a running container                                   |
+| [kill](kill.md) | Kill one or more running containers                        |
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: I'm still in doubt if `one or more` makes it more readable in this overview (or in general). (I know it's the description that's currently display in the usage), but not sure what would be best to show here.

Maybe keep the old description for the overview, and leave the "one or more" to the detail page.

File: docs/reference/commandline/index.md [hunk 2]
```diff
@@ -62,20 +62,20 @@ read the [`dockerd`](dockerd.md) reference page.
...
+| [restart](restart.md) | Restart one or more containers                       |
 | [rm](rm.md) | Remove one or more containers                                  |
 | [run](run.md) | Run a command in a new container                             |
 | [start](start.md) | Start one or more stopped containers                     |
 | [stats](stats.md) | Display a live stream of container(s) resource usage  statistics |
-| [stop](stop.md) | Stop a running container                                   |
+| [stop](stop.md) | Stop one or more running containers                        |
 | [top](top.md) | Display the running processes of a container                 |
-| [unpause](unpause.md) | Unpause all processes within a container             |
+| [unpause](unpause.md) | Unpause all processes within one or more container   |
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: s/container/containers/ (if we go for "one or more")

File: docs/reference/commandline/index.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: @mstanleyjones wdyt? Do you have suggestions? I find the "one or more" everywhere a bit awkward.

Comment (mstanleyjones) [inline docs/reference/commandline/index.md]: Maybe add another column like "Single container only" with y/n?

Comment (thaJeztah) [inline docs/reference/commandline/index.md]: Tempted to say that's best left to the actual reference page. I feel this page is to get a quick overview of available commands; e.g. a command to remove a container. Clicking through, I'll learn that I can also remove multiple containers in one go, or force removing, or use a filter, etc.

**Comment by thaJeztah** [comment]
oh, looks like this needs a rebase now

**Comment by allencloud** [comment]
Thanks a lot. @thaJeztah rebased.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**